spec/User_Defined_Domain_Maps.tex

```diff
@@ -1,7 +0,0 @@
-\sekshun{User-Defined Domain Maps}
-\label{User_Defined_Domain_Maps}
-
-\begin{craychapel}
-This chapter is forthcoming.  As a placeholder, please refer to
-doc/technotes/README.dsi in the Chapel release.
-\end{craychapel}
```

Comment:
I would like this section to stay.  I think User Defined Domain Maps should be defined in the spec (as well as the user's guide when we have one) and having an empty section for it reminds us to do it.